### DISCUSSION TITLE
Is Firebase Cloud Firestore integration possible ?

### DISCUSSION THREAD

**Discussion by labjte** [greenelab/lab-website-template#196] [Q&A] [answered]
i want to add some kind of database in the static website so i search around and found someone recommending using Firebase as a Database, is it possible to integrate it to this template ? sorry if this doesnt make any sense im fairly new at this

**Comment by vincerubinetti** [comment, marked as answer]
Unfortunately this is outside the scope of what I can help you with here, but I can say some general things about this.

This template is ultimately just a website, and you can provide whatever custom HTML CSS and JavaScript for it that you want. You could use Firestore sure, but you'd need to write some JavaScript and possibly other things to have the website interface with your database. There might be some third-party libraries that could help you a little bit, but you'd still need to know something about JavaScript to integrate something like this.

What are you actually trying to do?
